static/app/components/events/eventReplay/index.tsx

```diff
@@ -1,26 +1,57 @@
+import {useState} from 'react';
+
 import Button from 'sentry/components/button';
 import ErrorBoundary from 'sentry/components/errorBoundary';
 import EventDataSection from 'sentry/components/events/eventDataSection';
 import LazyLoad from 'sentry/components/lazyLoad';
+import {relativeTimeInMs} from 'sentry/components/replays/utils';
 import {t} from 'sentry/locale';
 
 type Props = {
   orgSlug: string;
   projectSlug: string;
   replayId: string;
+  eventTimestamp?: string;
```

Comment:
when is the new prop optional? looks like there's only one callsite. whatever is passed should be passed 100% of the time.